### PR TITLE
Port JMS Austocloseable FAT from WS-CD-Open and enable jakartaee

### PR DESCRIPTION
**Pull request by anjumfatima90**
Ref: #11851

Running the FAT for ee9 only.

### CODE REVIEW

File: dev/com.ibm.ws.messaging.open_jms20AutoCloseable_fat/publish/files/Qclose.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+    <variable name="onError" value="FAIL"/>
+    
+    <logging 
+  traceSpecification="*=info:logservice=all:ConnLeakLogic=all:WAS.j2c=all:SIBJms*=all"
+  traceFileName="Qclosetrace.log"
+  maxFileSize="200"
+    />
+
+    <messagingEngine >
```
Comment (dazavala) [inline dev/com.ibm.ws.messaging.open_jms20AutoCloseable_fat/publish/files/Qclose.xml:21]: Hard to read these configuration elements.  Clean up the formatting.

**Review by dazavala — CHANGES_REQUESTED** [review]
Hello, @anjumfatima90 .  Everything looks good except for some formatting which is inconsistent with other jms20 FATs and compromises readability in a few spots.  Please clean up the sources — remove blank lines where possible, fix the few lines I noted in the PR, modify the test classes to have similar format over all test methods.  If you stack a commit with the formatting changes, I can verify the second commit did not change test behaviors or configurations.  Thank you.

**Review by dazavala — APPROVED** [review]

**Comment by anjumfatima90** [comment]
#build

**Comment by anjumfatima90** [comment]
#libby

**Pull request merged**